### PR TITLE
fix doc

### PR DESCRIPTION
**Pull request by teto**

### CODE REVIEW

File: ci/file-list.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+www.neovim.io/doc2/lsp/index.html
```
Comment (jamessan) [inline ci/file-list.txt:1]: Isn't it just `neovim.io`?

**Review by jamessan — COMMENTED** [review]
Maybe it's better to have CI make a shallow clone of neovim/neovim.github.io and `cp -a doc2/*.html .` instead of running this script.

**Comment by jamessan** [comment]
FWIW, `docs` sounds better than `doc2`.

**Comment by teto** [comment]
A shallow clone does sound better. Alternatively, could we push from the website to this repo ?

**Comment by jamessan** [comment]
A pull model is better, in terms of permissions.

**Comment by teto** [comment]
actually a clone doesn't work because files in the clone dont have the same hierarchy as in the generated website (or I would need to generate the website after the clone).

File: ci/file-list.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+www.neovim.io/doc2/general/index.html
+www.neovim.io/doc2/treesitter/index.html
+www.neovim.io/doc2/lua-resources/index.html
```
Comment (jamessan) [inline ci/file-list.txt:4]:
```suggestion
https://neovim.io/doc2/lsp/index.html
https://neovim.io/doc2/general/index.html
https://neovim.io/doc2/treesitter/index.html
https://neovim.io/doc2/lua-resources/index.html
```

File: ci/doc-index.sh [hunk 3]
```diff
@@ -4,15 +4,14 @@ set -e
 set -u
 
 readonly BUILD_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
-readonly DOC_INDEX_PAGE_URL="${DOC_INDEX_PAGE_URL:-https://neovim.io/doc_index}"
 readonly DOC_SUBTREE='index.html'
```
Comment (jamessan) [inline ci/doc-index.sh:7]: This can be removed too

**Comment by teto** [comment]
This should be good to squash.

**Pull request merged**